package.json

```diff
@@ -33,7 +33,7 @@
     "fh-cls-mongoose": "2.1.1",
     "fh-cluster": "0.3.0",
     "fh-component-metrics": "2.7.0",
-    "fh-config": "2.0.0",
+    "fh-config": "1.0.3",
```

Comment:
Let's use "1.1.0" which the latest version before the release of the 2.0.0 version (gave it a try and it worked as well)

Reply:
@odra we come back for the latest version used and tested in this project before the change in order to ensure that no new issues will be introduced in the 4.7. Is it make sense? We should use the 2.0.0 with all the fixes as soon as possible. Wdyf? 

@grdryn would you like that we use the version 1.1.0 as suggested? Wdyf?